packages/cozy-harvest-lib/src/components/cards/__snapshots__/WebsiteLinkCard.spec.jsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`WebsiteLinkCard should render 1`] = `
+<withI18n(WebsiteLinkCard)
+  link="https://fc.assure.ameli.fr/FRCO-app/login"
+/>
+`;
```

Comment:
Not sure this is the withI18n we want to snapshot. Not very useful I think. Snapshoting WebsiteLinkCard will be better

Reply:
This is a copy from some other harvest unit test. It allow me to force a locale. It seems that passing t as a property in the unit test is not enough.

Reply:
Let's do that in an other way. 

Remove the translate() HOC and use our new useI18n() hook. (If we can use our new method in our new component, it's better ;)) 

Your component will be: 
```
const WebsiteLinkCard = ({ link}) => {
  const linkStyle = { textTransform: 'lowercase' }
  const label = new URL(link).host
  const { t } = useI18n()
  return (
    <Card>
      <SubTitle className="u-mb-1">{t('card.websiteLink.title')}</SubTitle>
      <hr className="u-silver" />
      <Media align="top">
        <Img className="u-pr-1">
          <Icon icon="globe" color={palette['coolGrey']} />
        </Img>
        <Bd>
          <ButtonLink
            subtle
            target="_blank"
            href={link}
            label={label}
            theme="text"
            className="u-dodgerBlue u-m-0"
            style={linkStyle}
          />
          <Caption>{t('card.websiteLink.description')}</Caption>
        </Bd>
      </Media>
    </Card>
  )
}
```

Then I will only test a few things like : 
```
const wrapper = mount(
      <I18n dictRequire={() => enLocale} lang="en">
        <WebsiteLinkCard link="https://fc.assure.ameli.fr/FRCO-app/login" />
      </I18n>
    )
    const component = wrapper.find(WebsiteLinkCard)
    expect(component.find('ButtonLink').props().href).toEqual(
      'https://fc.assure.ameli.fr/FRCO-app/login'
    )
    expect(component.find('ButtonLink').props().label).toEqual(
      'fc.assure.ameli.fr'
    )
```

to be sure that we have the right href & label.


Having a snapshot with `<withI18n(WebsiteLinkCard)
  link="https://fc.assure.ameli.fr/FRCO-app/login"
/>` is not wanted.

If you really want to have the full snapshoted component you can either mount() and then component.debug().match... or use a serializer like https://github.com/adriantoine/enzyme-to-json or something similar